--- process.py	2020-12-24 20:35:11.578947670 +0000
+++ process.py.modified	2020-12-24 20:38:57.777076157 +0000
@@ -491,6 +491,13 @@
         return self.__nmap_command_line or ""
 
     @property
+    def process_id(self):
+        """
+        Added by Ian Tibble, return unix pid for nmap process, allows it to be cancelled by user
+        """
+        return self.__nmap_proc.pid
+
+    @property
     def targets(self):
         """
         Provides the list of targets to scan
